airflow/executors/celery_executor.py

```diff
@@ -71,9 +71,7 @@
 @app.task
 def execute_command(command_to_exec: CommandType) -> None:
     """Executes command."""
-    if command_to_exec[0:3] != ["airflow", "tasks", "run"]:
-        raise ValueError('The command must start with ["airflow", "tasks", "run"].')
-
+    BaseExecutor.validate_command(command_to_exec)
```

Comment:
```suggestion
    cls.validate_command(command_to_exec)
```

that should work 🤷  - WDYT ?

Reply:
I think the `cls` will be undefined in this place

Reply:
Oh yes, sry didnt realize it was not a part of the class

Reply:
But I updated other usages to `self.` instead of `BaseExecutor.`